build.gradle

```diff
@@ -110,7 +110,7 @@ jenkinsPlugin {
     displayName = 'Jenkins Checkmarx Plugin'
 
     // URL for plugin on Jenkins wiki or elsewhere
-    url = 'https://wiki.jenkins-ci.org/display/JENKINS/Checkmarx+CxSAST+Plugin'
+    url = 'https://plugins.jenkins.io/checkmarx/'
```

Comment:
plugins.jenkins.io as URL is always wrong: The plugin needs to point so somwhere else, so the plugins site can pick that up as a reference.

Reply:
Thanks.  Reverted that change.  better to keep this pull request to a single commit.